core/tests/test_utils.py

```diff
@@ -482,6 +482,28 @@ def get_expected_logout_url(self, slug):
         """Returns the expected logout URL."""
         return current_user_services.create_logout_url(slug)
 
+    def get_html(self, url, params=None, expect_errors=False,
+                 expected_status_int=200):
+        """Get a HTML response, transformed to a Python object."""
+        html_response = self.testapp.get(
+            url, params, expect_errors=expect_errors,
+            status=expected_status_int)
+
+        # Testapp takes in a status parameter which is the expected status of
+        # the response. However this expected status is verified only when
+        # expect_errors=False. For other situations we need to explicitly check
+        # the status.
+        # Reference URL:
+        # https://github.com/Pylons/webtest/blob/
+        # bf77326420b628c9ea5431432c7e171f88c5d874/webtest/app.py#L1119 .
+        self.assertEqual(html_response.status_int, expected_status_int)
+        if not expect_errors:
```

Comment:
expect_errors is only for status checks starting with 4 or 5.

Note that you should also check the converse, i.e. that if expect_errors is True then the status_int starts with 4 or 5.

Reply:
done